content/en/tracing/setup_overview/custom_instrumentation/python.md

```diff
@@ -80,6 +80,9 @@ def make_sandwich_request(request):
 def make_sandwich_request(request):
     # Capture both operations in a span
     with tracer.trace("sandwich.create") as outer_span:
+        outer_span.resource = "resource_name"
```

Comment:
Should we instead do:

```python
with tracer.trace("sandwich.create", service="sandwich_service", resource="resource_name") as outer_span:
```

or are we trying to showcase that it can be updated via the properties?

(although we do show it before in `make_sandwich_request` ?

Reply:
Thanks for pointing that out, I missed that `tracer.trace` provided a way to specify this. I updated the example!